### PR TITLE
Run codespell on code

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/system/015-help.bats [hunk 1]
```diff
@@ -136,7 +136,7 @@ function check_help() {
             run_podman '?' "$@" $cmd </dev/null
             is "$status" 125 \
                "'$usage' indicates at least one required arg. I invoked it with no args and expected an error exit code"
-            is "$output" "Error:.* \(require\|specif\|must\|provide\|need\|choose\|accepts\)" \
+            is "$output" "Error:.* \(require\|specify\|must\|provide\|need\|choose\|accepts\)" \
```
Comment (edsantiago) [inline test/system/015-help.bats:139]: This one needs to revert: error messages can be "specify", "specified", or other possibilities.

**Comment by Luap99** [comment]
If you run these so often I wonder why do we not put this into a CI check?

**Comment by rhatdan** [comment]
No over time it has gotten much better. so we might want to do it automatically.

**Comment by rhatdan** [comment]
This runs with a clean
make codespell, so it could be added to the verify pass.

File: test/system/015-help.bats (hunk 1 above)
Comment (edsantiago) [reply test/system/015-help.bats:139]: /me thinks you forgot to revert this. Tests are gonna fail again.

**Pull request merged**